src/gobject/auto/param_spec.rs

```diff
@@ -0,0 +1,99 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+#[cfg(any(feature = "v2_38", feature = "dox"))]
+use Value;
+use gobject_ffi as ffi;
+use ffi as glib_ffi;
+use gobject_ffi;
+use object::IsA;
+use std::mem;
+use std::ptr;
+use translate::*;
+
+glib_wrapper! {
+    pub struct ParamSpec(Object<ffi::GParamSpec, ffi::GParamSpecClass>);
+
+    match fn {
+        get_type => || ffi::intern(),
```

Comment:
We need a way to override the get_type functions. This one here has a constant, `G_TYPE_PARAMSPEC`

Reply:
Agree